docs/basics/101-105-install.rst

```diff
@@ -70,15 +70,14 @@ can be a URL or a path to a local directory, or an SSH server [#f1]_. The datase
 to be installed lives on :term:`GitHub`, at
 `https://github.com/datalad-datasets/longnow-podcasts.git <https://github.com/datalad-datasets/longnow-podcasts.git>`_,
 and we can give its GitHub URL as the  first positional argument.
-Optionally, the command also takes a second positional path as an argument --
-to the *destination*, i.e., a path to where we want to install the dataset to --
-in this case it is ``recordings/longnow``.
+Optionally, the command also takes as second positional argument a path to the *destination*,
+--a path to where we want to install the dataset to-- In this case it is ``recordings/longnow``.
```

Comment:
Yep, sounds nicer! I have added a missing space, and was unsure about the second space followed by a capitalization, so I have changed it to a full stop.

```suggestion
-- a path to where we want to install the dataset to. In this case it is ``recordings/longnow``.
```